### ISSUE TITLE
Error when building blog

### ISSUE THREAD

**Issue by teonbrooks** [josh-collinsworth/sveltekit-blog-starter#17]
First, I want to say thanks so much for your blog post and this blog starter. It's really helped me with learning more about svelte and sveltekit.

I was trying to build the website but I keep running into this problem below. I've tried troubleshooting but I'm not exactly sure what's going wrong here. Do you have any insights?

```
  301 /blog/page -> /blog
file:///Users/teonbrooks/codespace/_websites/sveltekit-blog-starter/node_modules/@sveltejs/kit/src/core/prerender/prerender.js:429
		throw new Error(
		      ^

Error: The following routes were marked as prerenderable, but were not prerendered:
  - api/posts/category/[category],  - api/posts/page/[page],  - blog/category/page/[page],  - blog/page/[page],  - blog/category/[category]/page,  - blog/category/[category]/page/[page]

See https://kit.svelte.dev/docs/page-options#prerender-troubleshooting for more info
    at prerender (file:///Users/teonbrooks/codespace/_websites/sveltekit-blog-starter/node_modules/@sveltejs/kit/src/core/prerender/prerender.js:429:9)
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)

Node.js v18.0.0
[vite-plugin-svelte-kit] Prerendering failed with code 1
error during build:
Error: Prerendering failed with code 1
    at ChildProcess.<anonymous> (file:///Users/teonbrooks/codespace/_websites/sveltekit-blog-starter/node_modules/@sveltejs/kit/src/exports/vite/index.js:438:15)
    at ChildProcess.emit (node:events:527:28)
    at ChildProcess._handle.onexit (node:internal/child_process:291:12)
```

**Comment by josh-collinsworth** [comment]
Hey @teonbrooks – thanks for reporting! I think this is some combination of bad code I introduced by accident, and updated SvelteKit code that broke things. The short version is: I needed to add `fetch` to some `load` function arguments, and I needed to explicitly add all prerendered routes to `svelte.config.js`. (There was also a bit of logic adjustment and a slight refactor on how one of the `/api` routes works, too. Seems the root was that the `/api/blog/category/*` route was trying to do too much previously.)

Anyway, I've just pushed a commit that _should_ get the build succeeding again. Give it a shot and let me know if you're still running into issues. 🙏
